engine.py

```diff
@@ -53,8 +53,15 @@ def pre_app_init(self):
             # specifically for 2013 which is not officially supported but may work, output a warning:
             self.log_warning("This version of 3ds Max is not officially supported by Toolkit and you may "
                              "experience instability.  Please contact support@shotgunsoftware.com "
-                             "if you do have any issues.")        
-        
+                             "if you do have any issues.")
+
+        try:
+            max_ver_str = ".".join([str(v) for v in mxs.maxVersion()])
```

Comment:
I believe this is going to return something like 16000 (for Max 2014). Have a look at _max_version_to_year() in the maxplus engine. You'll probably want to copy that over and use it to convert what mxs.maxVersion() gives you into a "year" release number.